gandalf.js

```diff
@@ -5,56 +5,54 @@
  * for the extras/ plugins if they are loaded into a presentation.
```

Comment:
SCRIPT-There seems to be a typo in the function name. Did you mean `calculateAverage` instead of `calculateAverge`?